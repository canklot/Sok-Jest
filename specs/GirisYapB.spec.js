import TestComponent from '../components/GirisYapB.vue'
import { mount, RouterLinkStub } from "@vue/test-utils";
import UyeGiris from '../pages/exercise-2';
import { nextTick } from 'vue';


test('Giris yapmadan devam et tusu calisiyor mu', async () => {
  const wrapper = mount(UyeGiris);
  const form = wrapper.find('form'); /* form elementini bul */
  const input = form.find('GiriÅŸ yapmadan devam et'); /* Girisyapi bul tikla */
  form.trigger('submit');
  await nextTick();
  expect(wrapper).toMatchSnapshot();
});

test('mount a vue component', () => {
    const wrapper = mount(TestComponent,{ 
      stubs: {
        NuxtLink: RouterLinkStub
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
