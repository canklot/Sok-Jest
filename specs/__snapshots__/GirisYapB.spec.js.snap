// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Giris yapmadan devam et tusu calisiyor mu 1`] = `
<div>
  <form><input type="text" placeholder="Giris yap"> <button type="submit">submit</button></form>
  <h2>Tasks</h2>
  <ul></ul>
</div>
`;

exports[`mount a vue component 1`] = `
<div>
  <div class="alan1"><img src="../images/image1.jpg"> <a><button class="buton1">Giriş Yap</button></a> <a><button class="buton2">Yeni Üyelik Oluştur</button></a> <a>
      <p class="metin1">Giriş yapmadan devam et</p>
    </a></div>
</div>
`;
